string id
int8 hp
int64 num_elims
int64 num_respawns
